python/tests/test_lowlevel.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
```

Comment:
why still bother supporting python 2.7?

Reply:
because i use it ;)

Reply:
<guess the followup question> ;)  no really.  what's there to gain in this case?

Reply:
huh, github ate half of my response.  anyway, I'd be very careful about still supporting 2.7 at this point to be honest.